packages/core/test/unit/segments/attributes/trace_id.test.js

```diff
@@ -0,0 +1,47 @@
+var assert = require('chai').assert;
```

Comment:
Can we create a test to verify trace header keys are case insensitive?

Reply:
Already have one [here](https://github.com/aws/aws-xray-sdk-node/pull/284/files/76671cbfa5ae532306a2c29820b67ac91b48cf4c#diff-6a36c6c7818cffd493bc93a85aebe9b4R33-R40) :)